<launch>
  <arg name="image_raw_topic" default="/real_camera"/>
  <arg name="vel_cmd_topic" default="/android_input_vel"/>
  <arg name="twist_cmd_topic" default="/vel_cmd"/>
  <arg name="signal_cmd_topic" default="/signal_vel"/>
  <arg name="twist_linear_x" default="1"/>

    <node name="signal_panel_recognition" pkg="signal_recognition" type="signal_panel_recognition.py" output="screen">
      <param name="vel_cmd_topic" value="$(arg vel_cmd_topic)"/>
      <param name="image_raw_topic" value="$(arg image_raw_topic)"/>
      <param name="twist_cmd_topic" value="$(arg twist_cmd_topic)"/>
      <param name="signal_cmd_topic" value="$(arg signal_cmd_topic)"/>
      <param name="twist_linear_x" value="$(arg twist_linear_x)"/>
    </node>

</launch>